pyvista/plotting/renderer.py

```diff
@@ -1516,6 +1553,8 @@ def deep_clean(self, render=False):
             del self.edl_pass
         if hasattr(self, '_box_object'):
             self.remove_bounding_box(render=render)
+        if self._shadow_pass is not None:
+            self._shadow_pass = None
```

Comment:
Shouldn't we call `self.disable_shadows()` (after removing the `raise` on duplicate disable) either here or in `clear()`?

Reply:
Pending your comments on the previous suggestions.

Reply:
Agreed.  Added.